scripts/s3/s3api/copy_object.py

```diff
@@ -79,7 +79,9 @@ async def execute_copy_object_workload(self):
             logger.info("Iteration %s is started...", self.iteration)
             try:
                 # Put object in bucket1
-                if not isinstance(self.object_size, dict):
+                if isinstance(self.object_size, list):
+                    file_size = self.object_size[self.iteration % len(self.object_size)]
```

Comment:
Better to use random generator as this will not exercise upper values.

Reply:
This is for object of fix sizes which are defined in list and object operations will execute on the defined fix object sizes in a list. It will iterate over all the objects in a list sequentially.